code/widgets/org.eclipse.scout.jswidgets.ui.html/src/main/js/desktopnotification/DesktopNotificationForm.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2021 BSI Business Systems Integration AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.html
+ *
+ * Contributors:
+ *     BSI Business Systems Integration AG - initial API and implementation
+ */
+
+import {Button, Form, models, scout, Status} from '@eclipse-scout/core';
+import DesktopNotificationFormModel from './DesktopNotificationFormModel';
+import DesktopNotification from '@eclipse-scout/core/src/desktop/notification/DesktopNotification';
+
+export default class DesktopForm extends Form {
```

Comment:
Wrong Name, should be DesktopNotificationForm

Reply:
yes